docs/user-guide/petset/bootstrapping/index.md

```diff
@@ -49,10 +49,10 @@ vm-1      1/1       Running    0          2m
 We can exec into one and install nginx
 
 ```shell
-$ kubectl exec vm-0 /bin/sh
-vm-0 # apt-get update
+$ kubectl exec -it vm-0 /bin/sh
+vm-0 # apt-get update && apt-get install nginx -y
```

Comment:
I usually don't pile stuff onto a line in examples just to make sure the user isn't missing a step. suggest leaving it split.